### ISSUE TITLE
Llama 2: Using in languages other than English.

### ISSUE THREAD

**Issue by egor-miasnikov** [facebookresearch/llama#406]
Why do you have a restriction on using Llama 2 in languages other than English?

**Comment by tanishqvaidya10** [comment]
If You read **[5.2 Limitations and Ethical Considerations](https://ai.meta.com/research/publications/llama-2-open-foundation-and-fine-tuned-chat-models/)** Topic on Page 34, of Llama 2 Research Paper, you will clearly understand why there is a restriction on using Llama 2 in languages other than English.
Llama 2 has struggled to gain proficiency in other languages due to limited amount of data.

**Comment by tanishqvaidya10** [comment]
# If You read [5.2 Limitations and Ethical Considerations](https://ai.meta.com/research/publications/llama-2-open-foundation-and-fine-tuned-chat-models/) Topic on Page 34, of Llama 2 Research Paper, you will clearly understand why there is a restriction on using Llama 2 in languages other than English.
Llama 2 has struggled to gain proficiency in other languages due to limited amount of data. # # # # #

**Comment by tanishqvaidya10** [comment]
If you Read Topic [5.2 Limitations and Ethical Considerations](https://ai.meta.com/research/publications/llama-2-open-foundation-and-fine-tuned-chat-models/) On page 34 you will clearly understand why there is restriction on using Llama 2 in languages other than English.
They have also mentioned that Llama 2 has struggled to gain proficiency in other languages due limited amount of data.

**Comment by jspisak** [comment]
The Llama models thus far have been mainly focused on the english language. We are looking at true multi-linguality for the future but for now there are a lot of community projects that fine tune Llama models to support languages beyond english. Thanks!

**Comment by BeginnerGitHub** [comment]
Hi,
Sorry to raise the question again but I would like to be sure I understand correctly: in the paragraph you mention it says this "_Consequently, the model’s performance in languages other than English remains fragile and should be used with caution._"

So I understand that we can use Llama 2 in languages other than English and that this use is not illegal, the only problem is that Llama2 is less efficient in languages other than English.

Now the problem is when I read other passages like this ([source](https://huggingface.co/meta-llama/Llama-2-70b-hf)) :
> "**Out-of-scope Uses** Use in any manner that violates applicable laws or regulations (including trade compliance laws).Use in languages other than English. Use in any other way that is prohibited by the Acceptable Use Policy and Licensing Agreement for Llama 2."

The phrase "_Use in languages other than English_" is mentioned alongside the phrases "_Use in any manner that violates applicable laws or regulations..._" and "_Use in any other way that is prohibited by the Acceptable Use Policy and Licensing Agreement for Llama 2_.” which mention two prohibited/forbidden and illegal things...

So I have the impression that it is said that the use of Llama 2 in languages other than English has the same status, namely that it is prohibited/forbidden and illegal.

Can you please clarify my understanding and dispel this "ambiguity"?
THANKS.

PS: Sorry for my bad English, I'm not a English speaker.

**Comment by jspisak** [comment]
I've also added this to the FAQ (last question): https://github.com/facebookresearch/llama/blob/main/FAQ.md

**Comment by BeginnerGitHub** [comment]
Thank you for your reply.

In fact I just wanted to know if the use of Llama 2 in a language other than English was authorized. I ask this because I used Llama 2 in other languages (like French for example) and the answers seemed satisfactory to me...

For example, I asked for a Python code in French and the answer was correct... I also tested the translations in several languages...

Generally speaking, is this use in other languages authorized?

THANKS.
